app/models/pydantic/creation_options.py

```diff
@@ -225,10 +225,10 @@ class VectorSourceCreationOptions(StrictBaseModel):
     def validate_source_uri(cls, v, values, **kwargs):
         if values.get("source_driver") == VectorDrivers.csv:
             assert len(v) >= 1, "CSV sources require at least one input file"
-        else:
+        elif values.get("source_driver") in [VectorDrivers.esrijson, VectorDrivers.shp, VectorDrivers.geojson_seq, VectorDrivers.geojson]:
```

Comment:
This is a little tricky, because I don't think we actually support multiple geodatabase/filegdbs, we just support appending a different file. But if a user actually sent in multiple filegdbs with layers from each, I think it'd break. We have no way to know which layers are from which files.

So I think we do still want to enforce one source URI per operation? (new version or append)